src/mitol/payment_gateway/api.py

```diff
@@ -79,6 +79,15 @@ def __init_subclass__(cls, *, gateway_class, **kwargs):
 
         cls._GATEWAYS[gateway_class] = cls
 
+    def find_gateway_class(func):
+        def _find_gateway_class(cls, payment_type, *args, **kwargs):
```

Comment:
Could you decorate this with [`functools.wraps`](https://docs.python.org/3/library/functools.html#functools.wraps)?

Reply:
Added that in c6c7be4004f7a94c688e7da67edb516617119597